### PR TITLE
[Hotfix] GPU Enum Fix

### PR DESCRIPTION
**Pull request by Advitya17**
This PR fixes the GPU enum to ensure the device is correctly accessed and assigned.

### CODE REVIEW

**Comment by Advitya17** [comment]
TODO: confirm testing requirements.

File: python/ml_wrappers/model/image_model_wrapper.py [hunk 1]
```diff
@@ -316,8 +316,10 @@ def _get_device(device: str) -> str:
        or type(device) == int
        or device is None):
         if device == Device.AUTO.value:
-            return (Device.CUDA.value if torch.cuda.is_available()
-                    else Device.CPU.value)
+            if torch.cuda.is_available():
```
Comment (imatiach-msft) [inline python/ml_wrappers/model/image_model_wrapper.py:319]: did you have a chance to look into why the tests did not catch this - and is it possible to write a test for this?

**Review by imatiach-msft — APPROVED** [review]

File: python/ml_wrappers/model/image_model_wrapper.py [hunk 2]
```diff
@@ -316,8 +316,10 @@ def _get_device(device: str) -> str:
...
+                return Device.CUDA.value
```
Comment (natalie-isak) [inline python/ml_wrappers/model/image_model_wrapper.py:320]: I wonder if we should just set this to CPU (and leave a TODO comment) until we finish configuring CUDA?

**Review by natalie-isak — APPROVED** [review]

**Comment by Advitya17** [comment]
Abandoning the PT in favor of #127 which will pull changes from here.

**Pull request closed without merging**